src/components/proposal_list/index.tsx

```diff
@@ -64,7 +53,7 @@ const ProposalList = (props: ProposalListProps) => {
                     <div className={classnames('mainContent', 'time')}>
                       <div className={classnames('voting')}>
                         <span className={classnames('clock')}>
-                          <img src={imageUrl} alt="" className={classnames('clockImage')} />
+                          <AccessTimeIcon className={classnames('clockImage')} />
                           &nbsp;
                         </span>
```

Comment:
instead of span + &nbsp; why not directly add a margin right to `.clockImage`? what do you think